dev/Generated/PersonPicture.properties.cpp

```diff
@@ -136,6 +137,17 @@ void PersonPictureProperties::EnsureProperties()
                 ValueHelper<winrt::ImageSource>::BoxedDefaultValue(),
                 winrt::PropertyChangedCallback(&OnPropertyChanged));
     }
+    if (!s_TemplateSettingsProperty)
```

Comment:
Didn't you make some generated goop for creating new DPs?

Reply:
Yes, this is all generated code.